### PR TITLE
feat: auto remove expired files

### PR DESCRIPTION
**Pull request by fengmk2**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

### CODE REVIEW

File: app/schedule/rotateByFile.js [hunk 1]
```diff
@@ -59,5 +63,39 @@ module.exports = function(app) {
     });
   }
 
+  // remove expired log files: xxx.log.YYYY-MM-DD
+  function* removeExpiredLogFiles(logdir, maxDays) {
+    const files = yield fs.readdir(logdir);
+    const expriedDate = moment().subtract(maxDays, 'days')
+      .utcOffset(8)
```
Comment (fengmk2) [inline app/schedule/rotateByFile.js]: 这里我得改改。

**Comment by fengmk2** [comment]
合并后发 1.0.0

**Comment by fengmk2** [comment]
windows 下跑 size ratote 很怪，文件明明存在，但是判断却是返回 false

image

**Comment by dead-horse** [comment]
额，那不管它了 +1

**Comment by fengmk2** [comment]
嗯，合并吧。发布1.0.0

**Comment by dead-horse** [comment]
1.0.0

**Pull request merged**